app/components/courses/list-item.hbs

```diff
@@ -0,0 +1,69 @@
+<tr
+  class="courses-list-item {{if (includes @course.id @coursesForRemovalConfirmation) "confirm-removal"}}"
+  data-test-courses-list-item
+>
+  <td class="text-left" colspan="8" data-test-course-title>
+    <LinkTo @route="course" @model={{@course}}>
+      {{@course.title}}
+      {{#if @course.externalId}}
+        ({{@course.externalId}})
+      {{/if}}
+    </LinkTo>
+  </td>
+  <td class="text-center hide-from-small-screen" colspan="1" data-test-level>
+    {{@course.level}}
+  </td>
+  <td class="text-center hide-from-small-screen" colspan="2" data-test-start-date>
+    {{moment-format @course.startDate "L"}}
+  </td>
+  <td class="text-center hide-from-small-screen" colspan="2" data-test-end-date>
+    {{moment-format @course.endDate "L"}}
+  </td>
+  <td class="text-right" colspan="2" data-test-status>
+    {{#if (includes @course.id @savingCourseIds)}}
+      <LoadingSpinner />
+    {{else}}
+      <PublicationStatus @item={{@course}} @showText={{media "isLaptopAndUp"}} @showIcon={{not (media "isLaptopAndUp")}} />
+      {{#if @course.locked}}
+        {{#if this.canUnlock}}
+          <button
+            type="button"
+            class="link-button"
+            title={{t "general.unlockCourse"}}
+            {{on "click" (perform this.unlockCourse)}}
```

Comment:
Could maybe be `{{on "click" (fn @unlockCourse @course)}}`?

Reply:
:heavy_check_mark: